### ISSUE TITLE
Error in CreatePayment and no working examples?

### ISSUE THREAD

**Issue by HugCoder** [PayEx/PayEx.EPi.Commerce.Payment#11]
In the ReadMe you have example code for  
`private void CreatePayment(Cart cart, PaymentMethodDto.PaymentMethodRow paymentMethod)`
In there you assign:
`payExPayment.PaymentMethodId = payExPayment.PaymentMethodId;`
should it be `payExPayment.PaymentMethodId = paymentMethod.PaymentMethodId;`?

Also I would like to request some actual working examples. The documentation for PayEx is just abysmal, horrible and beyond bad. I just want to be able to redirect to the payment process, get an OK or Error, handle it on my server, and be done with it. Instead I read about different approaches on direct vs. redirect, having to create `Cart`, `PayExPayment`, `PayExPaymentGateway`, `PaymentTransactionTypeManager`, `PaymentStatusManager`, `TransactionScope`... the list goes on. Then there's talk about having a Commerce server and a Front End separated. How can it be so complicated?
Having used Stripe before, I'm just just blown away by how bad the API PayEx provides seems to be.

Can you please provide a working example that simplifies and makes sense of the documentation? I just want to enable credit card payment, not build an empire.

**Comment by asharghi** [comment]
That's correct, it should be payExPayment.PaymentMethodId = paymentMethod.PaymentMethodId; in the CreatePayment method. Thanks for catching that. We are going to update the ReadMe with more working example, and I will correct that line too.

**Comment by HugCoder** [comment]
No problem, thanks for working on this project!
